specs/redux/locs/actions.spec.js

```diff
@@ -7,31 +7,35 @@ import {store} from '../../init'
 import isEthAddress from '../../../src/utils/isEthAddress'
 import LOCModel from '../../../src/models/LOCModel'
 
-const accounts = UserDAO.web3.eth.accounts
+const account = UserDAO.web3.eth.accounts[0]
 let address
 
 describe('LOCs actions', () => {
-  it('should propose new LOC', (done) => {
-    LOCsManagerDAO.newLOCWatch((locModel) => {
-      expect(locModel.locName).toEqual("Bob's Hard Workers")
-      address = locModel.address
-      done()
-    })
-
+  it('should propose new LOC', done => {
     const data = {
-      locName: "Bob's Hard Workers",
+      locName: '1484554656',
       website: 'www.ru',
       issueLimit: 1000,
       publishedHash: 'QmTeW79w7QQ6Npa3b1d5tANreCDxF2iDaAPsDvW6KtLmfB',
       expDate: 1484554656,
-      account: accounts[0]
+      account
     }
-    store.dispatch(actions.submitLOC(data, () => {
-    }))
+
+    LOCsManagerDAO.newLOCWatch((locModel) => {
+      address = locModel.getAddress()
+      try {
+        expect(locModel.name()).toEqual('1484554656')
+        done()
+      } catch (e) {
```

Comment:
**Smells really bad**
Explain in details why did you use here such try-catch construction?